src/runtime_src/core/pcie/emulation/hw_em/zynqu/shim_hwem.cpp

```diff
@@ -151,6 +151,7 @@ int ZYNQShim::xclLoadXclBin(const xclBin *header) {
   cmd->addArg(xclBinName.c_str());
   uint32_t length;
   uint8_t* buff = cmd->generateBuffer(&length);
+  delete cmd;
```

Comment:
Yes, indeed.  Let fix is properly and use std::unique_ptr.

Reply:
Good idea. I've made the change